de-lib/src/main/java/org/iplantc/de/apps/integration/client/view/propertyEditors/widgets/SelectionItemPropertyEditor.java

```diff
@@ -239,9 +239,9 @@ public void setSuppressEvent(boolean suppressEventFire) {
     ColumnModel<SelectionItem> createColumnModel() {
         List<ColumnConfig<SelectionItem, ?>> list = Lists.newArrayList();
         SelectionItemProperties props = GWT.create(SelectionItemProperties.class);
-        displayCol = new ColumnConfig<>(props.display(), 120, labels.singleSelectDisplayColumnHeader());
-        nameCol = new ColumnConfig<>(props.name(), 150, labels.singleSelectNameColumnHeader());
-        valueCol = new ColumnConfig<>(props.value(), 150, labels.singleSelectValueColumnHeader());
+        displayCol = new ColumnConfig<>(props.display(), 120, appearance.singleSelectDisplayColumnHeader());
+        nameCol = new ColumnConfig<>(props.name(), 150, appearance.singleSelectNameColumnHeader());
+        valueCol = new ColumnConfig<>(props.value(), 150, appearance.singleSelectValueColumnHeader());
```

Comment:
Yeah, when I first started the refactors I was strict about that, but then I started to get a little lax about it when I realized even my eyes were glazing over a bit when pre-reviewing my PRs 😄   But you are correct, I believe we want these values in the appearance.